staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5424,6 +5424,12 @@ type Secret struct {
 	// +optional
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
+	// Immutable field, if set, ensures that data stored in the Secret cannot
```

Comment:
Since comments become docs...

"Immutable ensures that data ..." (basically drop 'field')

Reply:
Document the default?

Reply:
Is this field updatable or only set at creation?

Reply:
Can this field change from false to true?

Reply:
Addressed all (here and in configmap).